code/modules/mob/living/simple_animal/guardian/guardiannaming.dm

```diff
@@ -1,103 +1,169 @@
-
+#define GUARDIANCOLOROVERLAY "color overlay" //Used with spritecolor var for guardians to determine what color to apply
+#define GUARDIANCOLORBASE "color base"
+#define GUARDIANCOLORBOTH "color both"
+#define GUARDIANCOLORNONE "color none"
 /datum/guardianname
 	var/prefixname = "Default" //the prefix the guardian uses for its name
 	var/suffixcolour = "Name" //the suffix the guardian uses for its name
-	var/parasiteicon = "techbase" //the icon of the guardian
+	var/parasitebaseicon = "techbase" //the icon of the guardian
+	var/parasiteoverlayicon = "techglowbase"
```

Comment:
They looked fine to me? The little fuckers move so fast that no one cares how they look, its mostly a blur of a bright color and white.

ALSO YOU PICKED THE COLORS, SO ITS YOUR FAULT THAT THE COLORS DONT MATCH. 

ALSO, WERE NOT DOING FOUR FUCKING OVERLAYS PER HOLO FOR "QUALITY", THATS DUMB AS SHIT.